src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends IllegalArgumentException {
```

Comment:
You can consider extending from the Exception class instead. After all, it is the parent class of all exception classes.